wrapper

```diff
@@ -133,7 +137,7 @@ else
 	mv $cmd $actn
 	echo "Command '$action': <$SENDER> -> $dest  piped to mailman after spam check, saved as $actn"
 	$MAILMAN_BIN "$action" "$dest" < $actn
-	#rm -f $cmd
+	rm -f $cmd
```

Comment:
Does this ever remove anything? Didn't `$cmd` just get moved to `$actn`?

Reply:
Looks like it's left over, fine to delete it